src/query/models/tag.go

```diff
@@ -191,8 +192,7 @@ func (t Tags) IDWithExcludes(excludeKeys ...string) uint64 {
 	return h.Sum64()
 }
 
-// TagsWithoutKeys returns only the tags which do not have the given keys
-func (t Tags) tagsSubSet(keys []string, include bool) Tags {
+func (t Tags) tagSubSet(keys []string, include bool) Tags {
```

Comment:
nit: slightly prefer `tagSubset` since a `subset` is a thing but a `sub set` isn't. Your call though